Makefile

```
@@ -698,3 +698,36 @@ endif
+docker_build_multiarch:
+	@echo "Building multi-arch OVMS images for linux/amd64,linux/s390x,linux/ppc64le..."
+	docker buildx build \
+		--platform linux/amd64,linux/s390x,linux/ppc64le \
+		--file "Dockerfile.$(DIST_OS)" \
+		$(BUILD_ARGS) \
+		--tag "$(OVMS_CPP_DOCKER_IMAGE):$(OVMS_CPP_IMAGE_TAG)" \
+		--push \
+		.
+
+.PHONY: docker_build_per_arch
+docker_build_per_arch:
```

checkmake (0.3.2)

[warning] Target body for "docker_build_per_arch" exceeds allowed length of 5 lines (18).

(maxbodylength)